wolox-training/nh-android
# News frontend

## News Frontend

### Summary
Implemented a list that shows the news for a logged user.
When user swipes down, a loading icon is shown.
User can swipe up to see more news.
Added a button to the bottom of the screen that disappears when user swipes up and reappears when user swipes down.

**Cards:** 
* Card: [10] https://trello.com/c/UPwX4ddC/10-news

### Test Plan
1) Log in with an existing user if not already logged in, for example using mail "mail@test.com" and password "password", a list of news should appear, as shown in Screenshot 1.
2) Swipe down, a loading icon should appear, as shown in Screenshot 2.
3) Swipe up to move through the list, the green button on the bottom of the screen should disappear, as shown in Screenshot 3. Button should reappear when swiping down.
4) Tap on a heart, its color should change to red, as shown in Screenshot 4.

### Screenshots

#### Screenshot 1
![news](https://cloud.githubusercontent.com/assets/14595211/10321734/a80c7c6c-6c50-11e5-89ac-078312d7d55f.png)

#### Screenshot 2
![reloading](https://cloud.githubusercontent.com/assets/14595211/10321771/d03ed482-6c50-11e5-985c-506aa85b19c7.png)

#### Screenshot 3
![scrolling](https://cloud.githubusercontent.com/assets/14595211/10321739/ab9121bc-6c50-11e5-88b1-1f6d48856442.png)

#### Screenshot 4
![heart-clicked](https://cloud.githubusercontent.com/assets/14595211/10322005/1a5ce6ca-6c52-11e5-90ac-d92a835edcee.png)
